docs/how-to-guides/faststore-api/extending-the-faststore-api.md

```diff
@@ -2,14 +2,20 @@
 
 Although the [FastStore API](/reference/api/faststore-api) provides a complete [GraphQL schema for ecommerce](/reference/api/queries), some stores may need to access other, more specific, information.
 
-For those cases, it is possible to extend the FastStore API schema, adding new data to the existing [queries](/reference/api/queries).
+You should not add requests to other APIs to your storefront since this may [impact site performance](/how-to-guides/faststore-api/fetching-api-data#best-practices-for-fetching-data). For those cases, it is possible to extend the FastStore API schema, adding new data to the existing [queries](/reference/api/queries).
 
 In this guide you will learn how to implement this in your project. You can also view a [summary](#summary) of the expected behavior and a [complete code example](#complete-code-example).
 
 :::info
 Once you have implemented the schema extension in your code, you can run a local [test with a GraphQL IDE](/how-to-guides/faststore-api/explore-the-faststore-api).
 :::
 
+## Before starting
+
+Before extending the schema, check whether the [GraphQL root object](https://graphql.org/learn/execution/#root-fields-resolvers) contains the information you need. If it does not, follow the steps below to [extend the schema](#step-by-step).
```

Comment:
Do you think it would be a good idea to publish this change with this link sort of as a placeholder until we publish our doc with the root object specs? Or this leads to wrong information?

Adding the example sounds great!